prov/usnic/src/usdf_cq.c

```diff
@@ -492,16 +523,20 @@ usdf_cq_read_common_soft(struct fid_cq *fcq, void *buf, size_t count,
 		return -FI_EOPNOTSUPP;
 	}
 
+	soft_cq_entry_len = sizeof(struct usdf_cq_soft_entry);
```

Comment:
see various notes in `usdf_cq_readerr_soft` that also apply here, esp. about changing to type-safe pointers and eliminating this `_len` variable.